spring-integration-stomp/src/test/java/org/springframework/integration/stomp/client/StompServerIntegrationTests.java

```diff
@@ -73,22 +73,21 @@
 
 	@BeforeAll
 	public static void setup() throws Exception {
-		int port = SocketUtils.findAvailableTcpPort(61613);
 		ConfigurationImpl configuration =
 				new ConfigurationImpl()
 						.setName("embedded-server")
 						.setPersistenceEnabled(false)
 						.setSecurityEnabled(false)
 						.setJMXManagementEnabled(false)
 						.setJournalDatasync(false)
-						.addAcceptorConfiguration("stomp", "tcp://127.0.0.1:" + port)
+						.addAcceptorConfiguration("stomp", "tcp://127.0.0.1:" + TransportConstants.DEFAULT_STOMP_PORT)
```

Comment:
It is better to use a flaky `SocketUtils` rather than hard coding to a single port if the component doesn't support `0`.

I'd prefer disabling (per your comment), if you want to remove it completely.

Reply:
There is no such a class in SF 6.0 any more. 
So, I wouldn't mind to watch this until it fails with the port conflict 😄